roles/github/tasks/main.yml

```diff
@@ -0,0 +1,88 @@
+---
+- name: Concatenate the public keys
+  set_fact:
+    list_of_users: "{{ users | join(',') }}"
+
+- name: Create new Github Team in {{ github_org_name }}, and add users {{ users }}
+  uri:
+    url: '{{ github_api_teams }}'
+    headers:
+      Accept: application/vnd.github.hellcat-preview+json
+    method: POST
+    user: '{{ github_api_username }}'
+    password: '{{ github_api_password }}'
+    force_basic_auth: yes
+    status_code: 201
+    body_format: json
+    body: '{
+             "name": "{{ team_name }}",
+             "description": "A great team",
```

Comment:
Can we please add parameter for this and maybe just default it to `the {{ team_name }} team`?